distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1552,6 +1571,130 @@ public void testIngestPlugin() throws Exception {
     }
 
     @Test
+    public void testMetacardIngestNetworkPlugin() throws Exception {
+
+        // ** MATCHERS **
+        // ** Used for validation **
+        final Matcher<Node> hasInitialDescription = hasMetacardAttributeXPath("description",
+                "Lombardi");
+        final Matcher<Node> hasInitialMetacardTags = hasMetacardAttributeXPath("metacard-tags",
+                "GAMMA",
+                "DELTA",
+                "FOXTROT",
+                "resource");
+        final Matcher<Node> hasNewDescription = hasMetacardAttributeXPath("description", "None");
+        final Matcher<Node> hasNewMetacardTags = hasMetacardAttributeXPath("metacard-tags",
+                "ALPHA",
+                "BRAVO",
+                "resource");
+        final Matcher<Node> hasNewSecurityAccessGroups = hasMetacardAttributeXPath(
+                "security.access-groups",
+                "FIFA",
+                "NFL");
+
+        // ** CONFIG **
+        // ** Setup the services that will conditionally add attributes **
+        final String factoryPid =
+                "org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin";
+        final String symbolicName = "catalog-plugin-metacardingest-network";
+        // ** IP rule broken - not getting what we expect **
+        final String clientIP = "0:0:0:0:0:0:0:1";
+        final String clientScheme = "https";
+
+        Map<String, Object> networkRule1Properties = getServiceManager().getMetatypeDefaults(
+                symbolicName,
+                factoryPid);
+
+        Map<String, Object> networkRule2Properties = Maps.newHashMap(networkRule1Properties);
+
+        List<Object> attributeAdjustmentsForRule1 = ImmutableList.builder()
```

Comment:
You should be able to skip the `build` call by doing `ImmutableList.of("security.access-groups = FIFA, NFL");`.